regolith/helpers/l_contactshelper.py

```diff
@@ -45,11 +46,17 @@ def subparser(subpi):
         "--notes",
         help='fragment (single argument only) to be found in the notes section of a contact'
     )
+    subpi.add_argument(
+        "-f",
+        "--find",
+        nargs="+",
+        help='Search this collection by giving key element pairs'
```

Comment:
I think "filter" is a better word than "Search".  We usually say "key-value pairs" too.